src/Element.cc

```diff
@@ -93,6 +94,31 @@ void Element::SetCopyChildren(bool _value)
   this->dataPtr->copyChildren = _value;
 }
 
+/////////////////////////////////////////////////
+void Element::SetExplicitlySetInFile(const bool _value)
+{
+  this->dataPtr->explicitlySetInFile = _value;
+
+  ElementPtr element_ptr = this->GetFirstElement();
```

Comment:
nit: We should use `camelCase` for `element_ptr` per the style [guide](https://ignitionrobotics.org/docs/all/contributing#style-guides)

Reply:
Addressed in: 52cd85f7c78a366d72b266a842c372c48f1588ce